### DISCUSSION TITLE
downloaditem totalbytes is 0

### DISCUSSION THREAD

**Discussion by Fantoom** [cefsharp/CefSharp#3832] [General]
.NET 5, wpf

I have created a simple class which implements IDownloadHandler interface, and when I download a file, it downloads the file, but the TotalBytes property of downloaditem is 0. so I cant show the progress of download

**Comment by amaitland** [comment]
How does Chrome behave when you download the same url? Please include the exact version you are using. 

If the response doesn't provide a content length header then the total bytes will be 0. This is a use case you should allow for as the header is optional. 

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Length
